ApplyToBecomeInternal/ApplyToBecomeInternal.Tests/Models/ApplicationForm/Sections/FurtherInformationSectionTests.cs

```diff
@@ -63,14 +66,59 @@ public void Constructor_Doesnt_Set_Conditional_Rows_Following_No_Answers()
 				new FormField("Is your school part of a federation?", "No"),
 				new FormField("Is the school supported by a foundation, trust or other body (e.g. parish council) that appoints foundation governors?", "No"),
 				new FormField("Does the school currently have an exemption from providing broadly Christian collective worship issued by the local Standing Committee on Religious Education (SACRE)?", "No"),
-				new FormField("Please provide a list of your main feeder schools", application.SchoolAdFeederSchools),
-				new LinkFormField("The school's Governing Body must have passed a resolution to apply to convert to academy status. Upload a copy of the school’s consent to converting and joining the trust.",
-					"Governing body resolution document", 
-					application.GoverningBodyConsentEvidenceDocumentLink),
+				new FormField("Please provide a list of your main feeder schools", application.SchoolAdFeederSchools),				
 				new FormField("Has an equalities impact assessment been carried out and considered by the governing body?", "No"),
 				new FormField("Do you want to add any further information?", "No")
 			};
 			
+			var formSection = new FurtherInformationSection(application);
+			formSection.Heading.Should().Be("Further information");
+			formSection.SubSections.Should().HaveCount(1);
+			formSection.SubSections.First().Heading.Should().Be("Additional details");
+			formSection.SubSections.First().Fields.Should().BeEquivalentTo(expectedFields);
+		}
+		[Fact]
+		public void Constructor_Doesnt_Set_Conditional_Rows_Following_Yes_Answers()
```

Comment:
Again, the test method name should probably be Constructor_Sets_Additional_Rows_Following_Yes_Answers()